### ISSUE TITLE
Same on Obj

### ISSUE THREAD

**Issue by sandip-narwade** [nickjanssen/PatrolJS#8]
I tried same code for OBJ but not coming

**Comment by sandip-narwade** [comment]
AnyOne Help me

**Comment by ZackMercury** [comment]
Useless issue, you did not specify anything you get. An error, or what?

**Comment by sandip-narwade** [comment]
@ZackMercury 
I have added two Car **(.obj file)** on the map. On single click red car is moving and On Double Click black car is moving.  The problem is, when I pressed single click  red car is moving while moving red car , I pressed double click now black car is moving but red car is stopping without reaching the end point. I want to move both car.  [See Video](https://youtu.be/97JEJjM15V8) and [Code](https://transfer.pcloud.com/download.html?code=5ZEicV7ZpW2TcIR29okZiGfLZjokQzEf5bwJeFkWeLX9kApnV4hWX)

![double car|662x472](upload://uiUAhU06l4Niu5vHUCgwHkR6WSS.jpg)
